src/xmerl_xpath_macros.hrl

```diff
@@ -0,0 +1,63 @@
+%% esaml - SAML for erlang
+%%
+%% Copyright (c) 2013, Alex Wilson and the University of Queensland
+%% All rights reserved.
+%%
+%% Distributed subject to the terms of the 2-clause BSD license, see
+%% the LICENSE file in the root of the distribution.
```

Comment:
I'm not sure we have to keep this last part of the comment verbatim. It is not referencing a file in this actual "distribution".

Also, [we need to provide the whole BSD 2-clause license text somewhere](https://choosealicense.com/licenses/bsd-2-clause/) (and make it obvious that it applies to this file and not the rest of the LGPL code in this library.

Reply:
Right, we should add LGPL stuff to these files.

Also, we might not need the macro files. I'll see if it can be removed.